book/integration-service.md

```diff
@@ -205,6 +205,20 @@ The Integration service needs secrets mounted so that the `Environment Provision
 16. Repeat step 10
 17. Repeat step 11-13
 
+### Image extraction details
+
+Integration service is getting specific information about image being build by build-definition using individual functions. Each one of them corresponds with the results that are being reported by build-definitions.
+
+<ins>Results are following:</ins>
+ - IMAGE_URL
```

Comment:
Maybe we can have a short description for each result and then mention what each of them is used for - e.g. for this one we could say something like this:
```
Image repository where the built image was pushed. Used to construct the component image reference.
```